bundles/org.openhab.binding.benqprojector/src/main/java/org/openhab/binding/benqprojector/internal/discovery/MulticastListener.java

```diff
@@ -59,8 +57,9 @@ public class MulticastListener {
      */
     public MulticastListener(String ipv4Address) throws IOException, SocketException {
         InetAddress ifAddress = InetAddress.getByName(ipv4Address);
+        NetworkInterface netIF = NetworkInterface.getByInetAddress(ifAddress);
         logger.debug("Discovery job using address {} on network interface {}", ifAddress.getHostAddress(),
-                NetworkInterface.getByInetAddress(ifAddress).getName());
+                netIF != null ? netIF.getName() : "UNKNOWN");
```

Comment:
If you're interested in going one step further, two lines below:
`[WARNING] /home/jenkins/jenkins-agent1/workspace/PR-openHAB-Addons/bundles/org.openhab.binding.benqprojector/src/main/java/org/openhab/binding/benqprojector/internal/discovery/MulticastListener.java:[64,16] The method setInterface(java.net.InetAddress) from the type java.net.MulticastSocket is deprecated since version 14`

For reference: #14199

Reply:
Yeah, I saw those were deprecated. But I didn't immediately know the correct way to resolve it. Thanks for the reference. I'll have a look.

Reply:
But the fix should be in a separate PR, correct? Just in case you decide to back port this fix to 3.4.x.

Reply:
Didn't consider backporting, but good point. Let's keep the PRs small and tidy and merge this one as-is.